lib/components/molecules/AtomicCarousel/AtomicCarousel.js

```diff
@@ -0,0 +1,341 @@
+// @flow
+/**
+ *
+ * AtomicCarousel
+ *
+ */
+import React, { useState, useEffect, useCallback } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+
+import styles from './AtomicCarousel.style';
+import type { Props } from './types';
+import { ListVanilla } from '../List';
+import Button from '../../atoms/Button';
+import Icon from '../../atoms/Icon';
+import Heading from '../../atoms/Heading';
+import useInterval from './useInterval.hook';
+
+const AtomicCarousel = ({
+  dots,
```

Comment:
You still haven't reduced this interface. As discussed delegate this on compound components and that way you could avoid few of the booleans